03-task-manager/fizzbuzz.js

```diff
@@ -0,0 +1,27 @@
+let counter = 1;
```

Comment:
This looks great! Passes the tests for sure

Small nit: why do you need `counter` when you have `i` in the for loop?